kolibri/plugins/facility_management/assets/src/views/class-enroll-page/confirm-enrollment-modal.vue

```diff
@@ -42,7 +42,7 @@
       confirmEnrollment: 'Confirm enrollment of selected users',
       areYouSure: 'Are you sure you want to enroll the following users into {className}?',
```

Comment:
This would be an instance where the user-generated text (class name) should be [inside single quotes](https://github.com/learningequality/kolibri/issues/3021#issuecomment-367513435).

Does not fall strictly into the scope of this PR, but could be fixed easily. :slightly_smiling_face: